spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -217,12 +219,23 @@ def pack_measures(self):
         writer.save()
 
     def show_total_results(self):
+        """
+        Print total results to CLI
+        """
+
         printv('\n\nAveraged measures...', self.verbose, 'normal')
-        for stg, key in zip(['  Volume [mm^3] = ', '  (S-I) Length [mm] = ', '  Equivalent Diameter [mm] = '], ['volume [mm3]', 'length [mm]', 'max_equivalent_diameter [mm]']):
-            printv(stg + str(np.round(np.mean(self.measure_pd[key]), 2)) + '+/-' + str(np.round(np.std(self.measure_pd[key]), 2)), self.verbose, type='info')
 
-        printv('\nTotal volume = ' + str(np.round(np.sum(self.measure_pd['volume [mm3]']), 2)) + ' mm^3', self.verbose, 'info')
-        printv('Lesion count = ' + str(len(self.measure_pd['volume [mm3]'].values)), self.verbose, 'info')
+        for label, key in zip(['Volume [mm^3]', '(S-I) Length [mm]', 'Equivalent Diameter [mm]'], self.measure_keys):
```

Comment:
It looks like the linting error is because we now use the variable name `label`. However, we're also importing the `label` module:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/3e62a3310a35735635680aebcee0534f46a615f4/spinalcordtoolbox/scripts/sct_analyze_lesion.py#L15

So, I think we need to pick a different variable name here. Maybe `for measure_str, key in`?

(Although, we wouldn't need to use a second variable at all if https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4207/files#r1312014325 is implemented, too.)

Reply:
Resolved in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4207/commits/68683a37a939904149da14effa08f111da1283c0 --  no need for the second variable